docs/troubleshooting/clusterResourcePlacementApplied.md

```diff
@@ -0,0 +1,239 @@
+# How can I debug when my CRP ClusterResourcePlacementApplied condition is set to false?
+
+### Common scenarios:
+- When the CRP is unable to propagate resources to a selected cluster due to the resource already existing on the cluster and not being managed by the fleet controller.
+- When the CRP is unable to propagate resource to selected due to another CRP already managing the resource for selected cluster with a different apply strategy.
+- When the CRP is unable to propagate resource due to failing to apply manifest due to syntax errors or invalid resource configurations.

```

Comment:
For syntax errors it could only happen if the resource is being propagated through an envelope object, right?

Reply:
Yes